pkg/operation/etcdencryption/util_test.go

```diff
@@ -0,0 +1,31 @@
+package encryptionconfiguration
+
+import (
+	"testing"
+)
+
+func TestSliceElementCompare(t *testing.T) {
```

Comment:
BDD testing style

Reply:
removed util.go and util_test.go since the slice compare function was effectively removed